examples/package.json

```diff
@@ -12,6 +12,10 @@
   "dependencies": {
     "@headlessui/react": "^1.7.16",
     "@heroicons/react": "^2.0.18",
+    "@react-three/drei": "^9.80.2",
+    "@react-three/fiber": "^8.13.6",
+    "@types/react-color": "^3.0.6",
+    "@uiw/react-color": "^1.3.3",
```

Comment:
Why not use `react-color`? 

I don't know if this is related, but I noticed the color picker is off:
<img width="256" alt="Screenshot 2023-08-11 at 10 07 33 AM" src="https://github.com/drifting-in-space/y-sweet/assets/2099249/639230c9-fff8-41cc-aad3-33b6295ad0ea">

Reply:
`react-color` hasn't been touched in two years and causes noisy warnings with recent versions of react ([github issue](https://github.com/casesandberg/react-color/issues/888))

Reply:
oh, I see what you mean, the colors don't line up by column